src/IDF_Editor/IDFEditMain.bas

```diff
@@ -694,7 +694,7 @@ convUnits(59).siName = "eV"
 convUnits(60).siName = "percent"
 convUnits(61).siName = "percentage (as a real decimal)"
 convUnits(62).siName = "s"
-convUnits(63).siName = "W/m2 or deg C"
+convUnits(63).siName = "W/m2, deg C or cd/m2"
```

Comment:
@mjwitte @JasonGlazer I updated the units here in IDF_Editor and also updated all occurrences of 'W/m2 or deg C' to 'W/m2, deg C or cd/m2'. It seems like this setpoint2 field is the only place using it.